### PR TITLE
imagemagick: splitting out imagemagick-7 stream

### PR DESCRIPTION
**Pull request by MickeyPvX**
Fixes:
Splitting out versions for IM 7.x - IM 6.x also maintained, package is in enterprise-packages

Related: 
https://github.com/chainguard-dev/image-requests/issues/5094

### CODE REVIEW

**Review by kranurag7 — CHANGES_REQUESTED** [review]
I'll also change the subpackage name to `${{package.name}}-foo|bar` naming convention.

**Comment by MickeyPvX** [comment]
> I'll also change the subpackage name to `${{package.name}}-foo|bar` naming convention.

Ah yes, I did that in `imagemagick-6` but this one's older...updating...

**Review by kranurag7 — CHANGES_REQUESTED** [review]
In runtime dependency section of the subpackage, we can consider using the following:
```yaml
    dependencies:
      runtime:
        - ${{package.name}}
```

**Review by kranurag7 — DISMISSED** [review]

**Review by EyeCantCU — CHANGES_REQUESTED** [review]

File: imagemagick-7.yaml [hunk 1]
```diff
@@ -58,26 +61,26 @@ pipeline:
   - uses: strip
 
 subpackages:
-  - name: imagemagick-doc
+  - name: ${{package.name}}-doc
     description: "ImageMagick documentation and manpages"
     pipeline:
```
[the remark this answers is not in the document]
Comment (MickeyPvX) [reply imagemagick-7.yaml:66]: I assume this is probably needed over in `imagemagick-6` as well?

[the remark this answers is not in the document]
Comment (EyeCantCU) [reply imagemagick-7.yaml:69]: Exactky

**Review by EyeCantCU — APPROVED** [review]

**Pull request merged**